lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -302,103 +304,132 @@ def test_basic_mean_predictor_bad_value(self):
         self.assertAlmostEqual(result, plugin.BAD_VALUE)
 
 
-class Test_crps_minimiser_wrapper(IrisTest):
+class SetupInputs(SetupCubes):
+
+    """Create a class for setting up cubes for testing."""
+
+    @ManageWarnings(
+        ignored_messages=["Collapsing a non-contiguous coordinate."],
+        warning_types=[UserWarning])
+    def setUp(self):
+        """Set up expected output."""
+        super().setUp()
+        self.temperature_forecast_predictor_mean = (
+            self.historic_temperature_forecast_cube.collapsed(
+                "realization", iris.analysis.MEAN))
+        self.temperature_forecast_predictor_realizations = (
+            self.historic_temperature_forecast_cube.copy())
+        self.temperature_forecast_variance = (
+            self.historic_temperature_forecast_cube.collapsed(
+                "realization", iris.analysis.VARIANCE))
+
+        # Create a cube for testing wind speed.
+        self.wind_speed_forecast_predictor_mean = (
+            self.historic_wind_speed_forecast_cube.collapsed(
+                "realization", iris.analysis.MEAN))
+        self.wind_speed_forecast_predictor_realizations = (
+            self.historic_wind_speed_forecast_cube.copy())
+        self.wind_speed_forecast_variance = (
+            self.historic_wind_speed_forecast_cube.collapsed(
+                "realization", iris.analysis.VARIANCE))
+
+        self.initial_guess_for_mean = np.array([0, 1, 0, 1], dtype=np.float32)
+        self.initial_guess_for_realization = (
+            np.array([0, 1, 0, np.sqrt(1/3.), np.sqrt(1/3.), np.sqrt(1/3.)],
+                     dtype=np.float32))
```

Comment:
Initial guesses should be set up by default inside the minimisation plugin.  This is a mathematical choice and should not be left up to the user.

Reply:
This would need a bit more work, so is probably something to think about after this PR.